docker/compilers/gcc/Dockerfile

```diff
@@ -1,10 +1,19 @@
-FROM ubuntu:20.04 AS build
+ARG UBUNTU_VERSION=20.04
+
+ARG GCC_MAJOR_VERSION=12
```

Comment:
Why is this separated from the rest of gcc args?

Reply:
Because it affects both build and final image.

Reply:
Ah, got it, thanks